### PR TITLE
doc/releases: octopus draft notes

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
jenkins render docs

**Comment by liewegas** [comment]
jenkins render docs

File: doc/releases/octopus.rst [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+     ceph telemetry on
+
+   For more information about the telemetry module, see :ref:`the
+   documentation <telemetry>`.
+
+
+Upgrading from pre-Mimic releases (like Luminous)
+-------------------------------------------------
+
+You *must* first upgrade to Mimie (13.2.z) or Nautilus (14.2.z) before
```
Comment (yaarith) [inline doc/releases/octopus.rst]: Mimic

**Pull request merged**